locale/en/blog/community/node-v5.md

```diff
@@ -0,0 +1,54 @@
+---
+title: What You Should Know about Node.js v5 and More
+date: 2015-10-30T12:00:00.000Z
+status: publish
+category: Community
+slug: node-v5
+layout: blog-post.hbs
+---
+
+## There’s Something New with Node.js Releases
+
+We just released [Node.js v5.0.0](https://nodejs.org/en/blog/release/v5.0.0/). You might be thinking to yourself: These folks just released [Node.js v4.2.1](https://nodejs.org/en/blog/release/v4.2.1/) “Argon,” under the new Long Term Support (LTS) plan, now I need to download this? The answer is yes and no.
+
+Node.js is growing, and growing fast. As we continue to innovate quickly, we will focus on two different release lines. One release line will fall under our **LTS** plan. All release lines that have LTS support will be even numbers, and (most importantly) focus on stability and security. These release lines are for organizations with complex environments that find it cumbersome to continually upgrade. We recently released the first in this line: [Node.js v4.2.1](https://nodejs.org/en/blog/release/v4.2.1/) “Argon.”
+
+The other release line is called **Stable**. All release lines will be odd numbers, and have a shorter lifespan and more frequent updates to the code. The Stable release line will focus on active development of necessary features and refinement of existing APIs. Node.js version 5 is this type of release.
+
+We want to make sure that you are adopting the release that best meets your Node.js needs, so to break it down:
+
+Stay on or upgrade to Node.js v4.2.x if you need stability and have a complex production environment, e.g. you are a medium or large enterprise.
+
+Upgrade to Node.js v5.x if you have the ability to upgrade versions quickly and easily without distributing your environment.
```

Comment:
To fight having a Wall-Of-Text, this short paragraph and the previous one are good candidates to convert to bullets (especially given the colon ending the paragraph before that).